docs/sl/beginners/index.md

```diff
@@ -0,0 +1,66 @@
+- Internet, kjer imajo vsi dostop do odprtega finančnega sistema.
+- Internet, ki je zgrajen na nevtralni in odprti infrastrukturi, ki ni pod nadzorom nobenega podjetja ali posameznika.
+
+**Ethereum je bil predstavljen leta 2015 in je vodilna programabilna tehnologija veriženja blokov na svetu.**
+
+**Tako kot ostale tehnologije veriženja blokov ima tudi Ethereum svojo kriptovaluto, ki se imenuje Ether (ETH).** ETH je digitalni denar. Če ste že slišali za [bitcoin](http://bitcoin.org/), ima ETH veliko enakih lastnosti. Je v celoti digitalen in v trenutku ga je mogoče poslati komurkoli po vsem svetu. Količine ETH v obtoku ne določa nobena vlada ali podjetje – je decentralizirana in majhna. Ljudje po vsem svetu uporabljajo ETH za plačevanje, kot shrambo premoženja in kot jamstvo.
+
+**Za razliko od ostalih tehnologij veriženja blokov pa lahko Ethereum naredi mnogo več.** Ethereum je programabilen, kar pomeni, da ga lahko razvijalci uporabijo za [izdelavo novih vrst aplikacij](/use/#1-use-an-application-built-on-ethereum).
+
+Te decentralizirane aplikacije (ali "dappi") pridobijo prednosti tehnologij kriptovalut in veriženja blokov. Lahko so vredne zaupanja, kar pomeni, da bodo potem, ko so "naložene" v Ethereum, vedno delovale tako, kot so bile programirane. Nadzorujejo lahko digitalna sredstva in s tem ustvarijo nove vrste finančnih aplikacij. Lahko so decentralizirane, zaradi česar jih ne more nadzorovati nobena posamezna entiteta ali oseba.
+
+**Prav zdaj na tisoče razvijalcev po vsemu svetu razvija aplikacije na Ethereumu ter izumlja nove vrste aplikacij, od katerih lahko mnoge uporabite že danes:**
+
+- [**Denarnice za kriptovalute**](/use/#_3-what-is-a-wallet-and-which-one-should-i-use) omogočajo poceni in takojšnje plačevanje z ETH ali drugimi sredstvi
+- **Finančne aplikacije** omogočajo izposojo, posojanje ali vlaganje digitalnih sredstev
+- **Decentralizirani trgi** omogočajo trgovanje z digitalnimi sredstvi ali celo trgovanje z “napovedmi” o dogodkih v resničnem svetu
+- **Igre**, v katerih ste lastniki sredstev iz igre in lahko celo zaslužite pravi denar
+- **In še veliko,** veliko več.
+
+**Skupnost Ethereuma je največja in najbolj aktivna skupnost veriženja blokov na svetu.** Vključuje razvijalce jedra protokola, raziskovalce kriptoekonomije, cypherpunke, rudarske organizacije, imetnike ETH, razvijalce aplikacij, navadne uporabnike, anarhiste, podjetja na seznamu Fortune 500 in od zdaj naprej tudi **vas**.
+
+**Ethereum ni pod nadzorom nobenega podjetja ali centralizirane organizacije.** Ethereum vzdržuje in izboljšuje raznolika globalna skupnost, ki s svojim delom prispeva k vsemu, od jedra protokola do aplikacij za uporabnike. To spletno mesto je tako kot celoten projekt Ethereum postavila skupina ljudi, ki delajo za isti cilj.
+
+**Pozdravljeni v Ethereumu.**
+
+**Ne veste, kam naprej?**
+
+- Želite začeti uporabljati Ethereum? [ethereum.org/sl/use](/use/)
```

Comment:
These links should be updated to the equivalent Slovenian pages (i.e. /use/ --> /sl/use/)